crates/rolldown_binding/src/types/js_callback.rs

```diff
@@ -65,18 +65,20 @@ use rolldown_utils::debug::pretty_type_name;
 /// - Rust(Simplified): `MaybeAsyncJsCallback<(Option<String>, i32), Option<i32>>`
 /// - Js: `(a: string | null | undefined, b: number) => Promise<number | null | undefined | void> | number | null | undefined | void`
 /// - Js(Simplified): `(a: Nullable<string>, b: number) => MaybePromise<VoidNullable<number>>`
-pub type JsCallback<Args, Ret> = ThreadsafeFunction<Args, Either<Ret, UnknownReturnValue>, false>;
+pub type JsCallback<Args, Ret, const WEAK: bool = false> =
```

Comment:
```suggestion
pub type JsCallback<Args, Ret, const WEAK: bool = true> =
```

I think we could make it as the default.